instrumentation/opentelemetry_cowboy/src/opentelemetry_cowboy.erl

```diff
@@ -57,16 +57,22 @@ handle_event([cowboy, request, stop], Measurements, Meta, _Config) ->
                   {'http.response_content_length', maps:get(resp_body_length, Measurements)}
                  ],
     otel_span:set_attributes(Ctx, Attributes),
-    case Status of
+    StatusCode = transform_status_to_code(Status),
+    case StatusCode of
         undefined ->
-            {ErrorType, Error, Reason} = maps:get(error, Meta),
-            otel_span:add_event(Ctx, atom_to_binary(ErrorType, utf8), [{error, Error}, {reason, Reason}]),
-            otel_span:set_status(Ctx, opentelemetry:status(?OTEL_STATUS_ERROR, Reason));
-        Status when Status >= 400 ->
-            otel_span:set_attributes(Ctx, [{'http.status', Status}]),
+          case maps:get(error, Meta, undefined) of
+            {ErrorType, Error, Reason} ->
+              otel_span:add_event(Ctx, atom_to_binary(ErrorType, utf8), [{error, Error}, {reason, Reason}]),
```

Comment:
`atom_to_binary` should not be needed. That was an otel bug

Reply:
updated it after merge master 👍🏻